docker-compose.yaml

```diff
@@ -179,6 +181,65 @@ services:
     volumes:
       - ./cdr/cdm/jupyter/user_shared_workspace:/cdm_shared_workspace/user_shared_workspace
 
+  dev_jupyterhub:
```

Comment:
We probably only need one jupyterhub instance eventually and control permission based on different user groups. But I haven't completely sort it out yet.